components/gitpod-protocol/src/typings/globals.ts

```diff
@@ -11,5 +11,9 @@ interface Window {
         loggedUserID?: string;
```

Comment:
Should not we make @gitpod-io/team-gitpod owners to avoid bothering webapp each time?  @geropl @easyCZ fine?

Reply:
CODEOWNERS changed in this PR 🙏